src/MooLite/core/inventory/items/DrinkDetail.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionTypeHrid } from "src/MooLite/core/actions/ActionTypeHrid";
+import { ConsumableSlotTypeHrid } from "src/MooLite/core/inventory/ConsumableSlotTypeHrid";
+import { ItemHrid } from "src/MooLite/core/inventory/ItemHrid";
+
+export interface DrinkDetail {
```

Comment:
I think this should be called `CharacterConsumable`. I'm trying to mimic what I guess is the server structure. If it's static data, it is a `*Detail`, if it changes based on player state, it's `Character*`.